packages/nodes-base/nodes/SyncroMSP/v1/actions/customer/getAll/description.ts

```diff
@@ -0,0 +1,77 @@
+import {
+	CustomerProperties,
+} from '../../Interfaces';
+
+export const customerGetAllDescription: CustomerProperties = [
+	{
+		displayName: 'Limit',
+		name: 'perPage',
```

Comment:
Name it `limit` as well, even if the query string parameter is named `per_page`. This is what's used for most nodes.